constraint/pkg/client/drivers/k8scel/driver.go

```diff
@@ -193,11 +194,9 @@ func (d *Driver) Query(ctx context.Context, target string, constraints []*unstru
 			return nil, fmt.Errorf("unknown constraint template validator: %s", constraint.GetKind())
 		}
 
-		assumeVAPEnforcementNotDisabled := assumeVAPEnforcementWithDefault(constraint, VAPDefaultYes)
-
-		// if we assume VAP enforcement for a given constraint/template combo, Gatekeeper
+		// if we assume VAP enforcement for a given template, Gatekeeper
 		// should not be evaluating that constraint/template in an admission context.
-		if isAdmission && assumeVAPEnforcementNotDisabled && wrappedValidator.assumeVAPEnforcement {
+		if isAdmission && wrappedValidator.assumeVAPEnforcement {
```

Comment:
@JaydipGabani I think this goes away with the multi-enforcement action change?

Basically, we always enforce at the webhook level unless the constraint is configured not to.

Reply:
Good catch! missed this one.